core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -406,7 +406,11 @@ public void update(float delta) {
             // If player just stopped sneaking
             level.makeWalk();
         }
-        level.getPlayer().move(moveAngle);
```

Comment:
Can you also stop it from creating flares & sprinting/sneaking when they die? Ideally it would stop enemy movement too (so like the only thing that would happen is the death animation once the player is dying), but flares/sneaking/sprinting shouldn't be too hard

Reply:
Is there a way to freeze the world? This is just a hunch, but perhaps not calling world.step()? Not sure if that would break the death animation but something to try

Reply:
FYI that happens here: 
`world.step(timeStep, WORLD_VELOC, WORLD_POSIT);`
in function fixedStep in LevelController